packages/mux-player/src/template.ts

```diff
@@ -43,6 +43,14 @@ export const content = (props: MuxTemplateProps) => html`
       debug="${props.debug ?? false}"
       prefer-mse="${props.preferMse ?? false}"
       start-time="${props.startTime != null ? props.startTime : false}"
+      metadata-video-id="${props.metadataVideoId ?? props.metadata?.video_id ?? false}"
+      metadata-video-title="${props.metadataVideoTitle ?? props.metadata?.video_title ?? false}"
+      metadata-viewer-user-id="${props.metadataViewerUserId ?? props.metadata?.viewer_user_id ?? false}"
```

Comment:
this looks good but even better I think is using the same pattern we do with `getVideoAttribute` in the `metadata` getter and there wouldn't be a need for extra props in `getProps`. only use the `.metadata` property

Reply:
see above comment